files/get_files_checksums.sh

```diff
@@ -33,7 +33,7 @@ done > "$listfile"
 orig_conf="$firewall_conf_root/firewalld.conf"
 remove_firewall_conf=true
 if [ -f "$orig_conf" ]; then
-    if [ -z "$package" ] || rpm -V "$package" | grep -q "c ${orig_conf}$"; then
+    if [ -z "$package" ] || (rpm -V "$package" || true) | grep -q "c ${orig_conf}$"; then
```

Comment:
Is the problem here the `set -o pipefail` behavior?  You may have to turn it off e.g. on the line before this one do `set +o pipefail`, then after set it back to `set -o pipefail`